apps/nft-service/src/nft/ipfs.ts

```diff
@@ -31,7 +33,7 @@ export async function uploadAvatar(avatar: Buffer, signature: string) {
   // First, validate we are uploading something that is probably right.
   if (!await validateImage(avatar, signature))
     return null;
-  return upload(avatar);
+  return upload(avatar, "avatar.jpg");
```

Comment:
Is it ok if it overrides the last file imported?

Reply:
I tested it, and there are multiple results on the IPFS node.

I -think- it would be better to have a unique filename, but then nobody (other than us) should ever see these names.  And actually, I'm not sure being able to identify the origin of any given image is a good thing (like, maybe that's too much info?  Could then connect a users Eth account with their profile pic, then their real-world identity, then connect their eth spending etc.  In the end, it may actually be the better choice to strip away any other real-world info.

![image](https://user-images.githubusercontent.com/4876160/120345308-f6cb6900-c2bf-11eb-92e6-63fab7acb2aa.png)